### PR TITLE
Added ISO layout and my keymap to kbd67 rev1

### PR DESCRIPTION
**Pull request by FSund**
## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/kbdfans/kbd67/rev1/rev1.h [hunk 1]
```diff
@@ -55,3 +55,17 @@
     { K40, K41,   KC_NO, K43, KC_NO, KC_NO, K46, KC_NO, KC_NO, KC_NO, K4A, K4B, K4C,   K4D,   K4E,   K4F }, \
 }
 
+#define LAYOUT_65_iso( \
+    K00, K01, K02, K03, K04, K05, K06, K07, K08, K09, K0A, K0B, K0C,      K0E, K0F, \
+    K10,      K12, K13, K14, K15, K16, K17, K18, K19, K1A, K1B, K1C, K1D, K1E, K1F, \
+    K20,      K22, K23, K24, K25, K26, K27, K28, K29, K2A, K2B, K2C, K2D,      K2F, \
```
Comment (fauxpark) [inline keyboards/kbdfans/kbd67/rev1/rev1.h:61]: This layout is not correct. The backslash keycode should go before the Enter, not above it as it is in ANSI: https://github.com/qmk/qmk_firmware/blob/master/layouts/default/65_iso/default_65_iso/keymap.c

Please also add the layout to the info.json.

File: keyboards/kbdfans/kbd67/rev1/keymaps/fsund/config.h [hunk 2]
```diff
@@ -0,0 +1,19 @@
+/* Copyright 2018 MechMerlin
```
Comment (fauxpark) [inline keyboards/kbdfans/kbd67/rev1/keymaps/fsund/config.h:1]: This file can be deleted.

File: keyboards/kbdfans/kbd67/rev1/rev1.h (hunk 1 above)
Comment (noroadsleft) [inline keyboards/kbdfans/kbd67/rev1/rev1.h:61]:
```suggestion
    K10,      K12, K13, K14, K15, K16, K17, K18, K19, K1A, K1B, K1C, K1D,      K1F, \
    K20,      K22, K23, K24, K25, K26, K27, K28, K29, K2A, K2B, K2C, K1E, K2D, K2F, \
```

File: keyboards/kbdfans/kbd67/rev1/keymaps/fsund/keymap.c [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+#include QMK_KEYBOARD_H
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+
+[0] = LAYOUT_65_iso(
+    KC_ESC,  KC_1,    KC_2,    KC_3,    KC_4,    KC_5,    KC_6,    KC_7,    KC_8,    KC_9,    KC_0,    KC_MINS, KC_EQL,  KC_BSPC, KC_DEL,
+    KC_TAB,  KC_Q,    KC_W,    KC_E,    KC_R,    KC_T,    KC_Y,    KC_U,    KC_I,    KC_O,    KC_P,    KC_LBRC, KC_RBRC, KC_BSLS, KC_INS,
+    MO(1),   KC_A,    KC_S,    KC_D,    KC_F,    KC_G,    KC_H,    KC_J,    KC_K,    KC_L,    KC_SCLN, KC_QUOT,          KC_ENT,  KC_VOLU,
```
Comment (noroadsleft) [inline keyboards/kbdfans/kbd67/rev1/keymaps/fsund/keymap.c:23]:
```suggestion
    KC_TAB,  KC_Q,    KC_W,    KC_E,    KC_R,    KC_T,    KC_Y,    KC_U,    KC_I,    KC_O,    KC_P,    KC_LBRC, KC_RBRC,          KC_INS,
    MO(1),   KC_A,    KC_S,    KC_D,    KC_F,    KC_G,    KC_H,    KC_J,    KC_K,    KC_L,    KC_SCLN, KC_QUOT, KC_BSLS, KC_ENT,  KC_VOLU,
```

File: keyboards/kbdfans/kbd67/rev1/keymaps/fsund/keymap.c [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+    KC_LSFT, KC_NUBS, KC_Z,    KC_X,    KC_C,    KC_V,    KC_B,    KC_N,    KC_M,    KC_COMM, KC_DOT,  KC_SLSH, KC_RSFT, KC_UP,   KC_VOLD,
+    KC_LCTL, KC_LGUI, KC_LALT,                   KC_SPC,                             KC_RALT, MO(2),   KC_RCTL, KC_LEFT, KC_DOWN, KC_RGHT
+),
+
+[1] = LAYOUT_65_iso(
+    KC_GRV,  KC_F1,   KC_F2,   KC_F3,   KC_F4,   KC_F5,   KC_F6,   KC_F7,   KC_F8,   KC_F9,   KC_F10,  KC_F11,  KC_F12,  _______, KC_MPLY,
+    _______, _______, KC_UP,   _______, _______, _______, _______, _______, _______, _______, KC_PSCR, _______, _______, _______, KC_MUTE,
+    _______, KC_LEFT, KC_DOWN, KC_RGHT, _______, _______, _______, _______, _______, _______, _______, _______,          _______, KC_MPRV,
```
Comment (noroadsleft) [inline keyboards/kbdfans/kbd67/rev1/keymaps/fsund/keymap.c:31]:
```suggestion
    _______, _______, KC_UP,   _______, _______, _______, _______, _______, _______, _______, KC_PSCR, _______, _______,          KC_MUTE,
    _______, KC_LEFT, KC_DOWN, KC_RGHT, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, KC_MPRV,
```

File: keyboards/kbdfans/kbd67/rev1/keymaps/fsund/keymap.c [hunk 5]
```diff
@@ -0,0 +1,44 @@
...
+    _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, KC_PGUP, KC_MNXT,
+    _______, _______, _______,                   _______,                            _______, _______, _______, KC_HOME, KC_PGDN, KC_END
+),
+
+[2] = LAYOUT_65_iso(
+    RESET,   _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,
+    _______, RGB_TOG, RGB_MOD, RGB_HUI, RGB_SAI, RGB_VAI, _______, _______, _______, _______, _______, _______, _______, _______, _______,
+    _______, _______, _______, RGB_HUD, RGB_SAD, RGB_VAD, _______, _______, _______, _______, _______, _______,          _______, _______,
```
Comment (noroadsleft) [inline keyboards/kbdfans/kbd67/rev1/keymaps/fsund/keymap.c:39]:
```suggestion
    _______, RGB_TOG, RGB_MOD, RGB_HUI, RGB_SAI, RGB_VAI, _______, _______, _______, _______, _______, _______, _______,          _______,
    _______, _______, _______, RGB_HUD, RGB_SAD, RGB_VAD, _______, _______, _______, _______, _______, _______, _______, _______, _______,
```

**Pull request closed without merging**